src/components/graphs/PieChart.vue

```diff
@@ -180,4 +200,110 @@ export default class PieChart extends Vue {
     font-size: 0.65em;
   }
 }
+
+.tooltip {
+  align-self: flex-end;
+  z-index: 10000;
+  margin: 0rem 2rem 1rem 0rem;
+
+  .tooltip-inner {
+    background: $white;
+    color: $black;
+    border-radius: 4px;
+    border: 1px solid $blue-dark;
+    box-shadow: 4px 4px 4px $grey;
+    padding: 5px 10px 4px;
+    width: 22rem;
+    padding: 1rem;
+
+    h6 {
+      font-size: 1rem;
+      color: $blue-dark;
+      margin: 0;
+      margin-bottom: 0.75rem;
+    }
+  }
+
+  .tooltip-arrow {
+    width: 0;
+    height: 0;
+    border-style: solid;
+    position: absolute;
+    margin: 5px;
+    border-color: $blue-dark;
+    z-index: 1;
+  }
+
+  &[x-placement^="top"] {
```

Comment:
Can you document what these do? I think this could go into our global CSS so folks can use Vue tooltip anywhere and get your styles, and then you could document what the different options are.